packages/legacy/core/App/services/logger.ts

```diff
@@ -1,10 +1,12 @@
 import { BaseLogger } from '@credo-ts/core'
 import { consoleTransport, logger } from 'react-native-logs'
 
-export class ConsoleLogger extends BaseLogger {
+import { BifoldError } from '../types/error'
+
+export class BifoldLogger extends BaseLogger {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  private log: any
-  private config = {
+  _log: any
+  _config = {
```

Comment:
`private` fields cannot be overridden in extended classes, that's why these are now just prefixed with an underscore

Reply:
How do they get overridden? Do you need a setter or getter?